message_types/communication.proto

```diff
@@ -0,0 +1,169 @@
+syntax = "proto3";
+
+message Communication {
+    message ServerChannel {
+        message Lan {
+            bool value = 1;
+        }
+        // Todo add wifi rs & zigbee. Unclear which values.
```

Comment:
Wifi is 3

Reply:
```serverChannel {
  lan {
  }
  3 {
    1: 1
  }
  4: ""
  6: ""
}```

Is what I get when I decode this with my inverter set to WiFi mode

Reply:
I have added Wifi and changed "value" to "enabled".